client/api.go

```diff
@@ -190,6 +200,8 @@ type API struct {
 	TerminalLocations *terminallocation.Client
 	// TerminalReaders is the client used to invoke /terminal/readers related APIs.
 	TerminalReaders *terminalreader.Client
+	// ThreeDSecures is the client used to invoke /3d_secure related APIs.
+	ThreeDSecures *threedsecure.Client
```

Comment:
@remi-stripe Haha, I imagine that you're not super excited about exposing some of these. I kind of figured we should probably just do it for consistency as long as the subpackages are still present in the project. The only alternative option would be to add a blacklist in the check script, and it'd be nice if we could avoid that. Thoughts?

Reply:
Yep I started typing and then realized it's cleaner to have it and remove later.